kafka-ui-react-app/src/components/common/Search/__tests__/Search.spec.tsx

```diff
@@ -11,36 +10,36 @@ jest.mock('use-debounce', () => ({
 describe('Search', () => {
   const handleSearch = jest.fn();
   it('calls handleSearch on input', () => {
-    const component = mount(
-      <ThemeProvider theme={theme}>
-        <Search
-          handleSearch={handleSearch}
-          value=""
-          placeholder="Search bt the Topic name"
-        />
-      </ThemeProvider>
-    );
-    component.find('input').simulate('change', { target: { value: 'test' } });
-    expect(handleSearch).toHaveBeenCalledTimes(1);
-  });
-
-  describe('when placeholder is provided', () => {
-    const component = shallow(
+    const component = render(
       <Search
         handleSearch={handleSearch}
         value=""
         placeholder="Search bt the Topic name"
       />
     );
+    const input = component.baseElement.querySelector('input') as HTMLElement;
```

Comment:
In testing library we use `screen.(get|query|find)(ByRole/ByText)` to do those kind of queries

Reply:
If you have multiple nodes with same role - you have to add object as second param
`screen.getByRole('textbox', {name: 'Searchbox accessible name'})`
If it has label - you can add `htmlFor` to label
If it doesn't have accessible name - you can add `aria-label` and it's description

The point of testing library is to test your UI and to make it accessible
Those kind of queries point you to errors in accessibility and urge to improve it